specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-10-01/examples/ContainerGroupsCreateOrUpdate.json

```diff
@@ -8,6 +8,12 @@
       "id": "/subscriptions/subid/resourceGroups/demo/providers/Microsoft.ContainerInstance/containerGroups/demo1",
       "location": "west us",
       "name": "demo1",
+      "Identity": { 
```

Comment:
This example also needs to be updated after your last commit, see the failure in the model validator [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/436274448).